client-guide/heartbeat-meeting.html.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Heartbeat meeting
+meta_description: >
+  TODO
+---
+
+We aim to have this meeting quarterly. You will talk with our Team lead about our relationship.
+
+## Goals
+
+The main goals of this recurring event are:
+
+- understand if our relationship is strong;
+- understand what's not going well in the collaboration;
+- understand if you need any adjustment to the team (additional team members, particular skill sets, etc.) in the next period;
```

Comment:
*any adjustments

Also, there should be a dot at the end of this sentence, since it's the last item of the list.

I think we could also reword "in the next period", maybe "in the upcoming months"?